nil/services/rpc/jsonrpc/types.go

```diff
@@ -93,6 +94,60 @@ type DebugRPCBlock struct {
 	OutTransactions []hexutil.Bytes        `json:"outTransactions"`
 	Receipts        []hexutil.Bytes        `json:"receipts"`
 	Errors          map[common.Hash]string `json:"errors"`
+	Config          ChainConfig            `json:"config"`
```

Comment:
Let's use pointer here. Because this field is empty for non-main shard

Reply:
It's not empty, we fetch config from corresponding `block.MainChainHash`